readthedocs/settings/base.py

```diff
@@ -718,7 +718,8 @@ def DOCKER_LIMITS(self):
 
     # Organization settings
     RTD_ALLOW_ORGANIZATIONS = False
-    ORG_DEFAULT_SUBSCRIPTION_PLAN_SLUG = 'trial-v2-monthly'
+    RTD_ORG_DEFAULT_STRIPE_PRICE = 'trial-v2-monthly'
```

Comment:
```suggestion
    RTD_ORG_DEFAULT_SUBSCRIPTION_STRIPE_PRICE = 'trial-v2-monthly'
```

Shouldn't we add the "subscription" word here to make it more clear?